assets/css/style.scss

```diff
@@ -26,9 +27,8 @@ $hr-border-color: #eff0f1;
 
 @import "{{ site.theme }}";
 
-
 .btn {
-    color: #000;
-    border-width: 3px;
-    font-weight: bold;
-}
\ No newline at end of file
+  color: #000;
+  border-width: 3px;
+  font-weight: bold;
+}
```

Comment:
Maybe just a remnant, but I don't see btn referenced in the HTML or .njk files in this case. I could very well be overlooking it tho.

Reply:
This is the `style.scss` file from the original project! Once I move the `os_site` folder into the root directory, we can get rid of these.

Reply:
oh sure enough! Makes sense.